grc/gui/PropsDialog.py

```diff
@@ -215,25 +215,10 @@ def _update_docs_page(self):
         if not docstrings:
             return
 
-        # show documentation string from block xml
-        from_xml = docstrings.pop('', '')
-        for line in from_xml.splitlines():
-            if line.lstrip() == line and line.endswith(':'):
-                buf.insert_with_tags_by_name(pos, line + '\n', 'b')
-            else:
-                buf.insert(pos, line + '\n')
-        if from_xml:
-            buf.insert(pos, '\n')
-
-        # if given the current parameters an exact match can be made
-        block_constructor = self._block.templates.render('make').rsplit('.', 2)[-1]
-        block_class = block_constructor.partition('(')[0].strip()
-        if block_class in docstrings:
-            docstrings = {block_class: docstrings[block_class]}
```

Comment:
After removing this part the loop below will display the same information as before. If we want to limit the display of the python docstrings to a specific class name we  should do that in the loop over the dict below instead of recreating a new dict again.

Reply:
Not for all types of blocks...

Creativing a small dict is pretty cheap.

Reply:
So, why do we need this here? I'm not against keeping this if there is a reason.
It has to be refactored to work with the `docstrings` dict passed in from `self._block.documentation`. Since we don't want to overwrite the reference.

Reply:
Let me think...hm...no, there is no reason at all....adding nonsense code is always just pure fun =)

e.g. the *_xx Blocks give duplicates. Other false positives are be hidden by this as well.
I don't see any reference being overwritten (this is not C++).

Reply:
I see, I'll rewrote the loop to match the previous behaviour. Or is the code ugly now? I checked it with the vector_sink/vector_insert to have the YAML parts & python docstrings properly displayed and it looks right, even if the type of the block is changed.